### ISSUE TITLE
HTTP Notifications are sent in doubles

### ISSUE THREAD

**Issue by MericFeyz** [ScorpioBroker/ScorpioBroker#388]
When I triggered the notification, I expected it to be just one notification, however I got two notifications which are identical apart from 
their notification IDs and the date they are sent, which differ by one second each. Following is the output of my http server: 
```
[~/GitProjects]$ python3 server/server.py
INFO:root:Starting httpd...

INFO:root:POST request,
Path: /
Headers:
user-agent: Vert.x-WebClient/4.2.7
Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJCYWZhVHdwMzFFMWl6RWdLUmItSGpaSWp2VDBRenZ1MmV3MjBnbE9abFJVIn0.eyJleHAiOjE2ODI2ODgxMTgsImlhdCI6MTY4MjY4NDUxOCwianRpIjoiMjZmMWJjYTYtMjkyOC00MTE0LWFmZTUtNDdkZTc2MmY1Y2MyIiwiaXNzIjoiaHR0cDovL2tleWNsb2FrLmxvY2FsL2F1dGgvcmVhbG1zL2lmZiIsImF1ZCI6WyJvaXNwLWZyb250ZW5kIiwicmVhbG0tbWFuYWdlbWVudCIsImFjY291bnQiXSwic3ViIjoiNTQyZjkyYmItNmY3Yy00ODVjLWJiMjEtOGUzZGExZWIxZDg3IiwidHlwIjoiQmVhcmVyIiwiYXpwIjoic2NvcnBpbyIsInNlc3Npb25fc3RhdGUiOiJlMzFlZjA0Zi1hZjViLTRhYmQtOGYxNi1iZGE1ZmQ4YmE4NjgiLCJyZWFsbV9hY2Nlc3MiOnsicm9sZXMiOlsib2ZmbGluZV9hY2Nlc3MiXX0sInJlc291cmNlX2FjY2VzcyI6eyJyZWFsbS1tYW5hZ2VtZW50Ijp7InJvbGVzIjpbIm1hbmFnZS11c2VycyJdfSwib2lzcC1mcm9udGVuZCI6eyJyb2xlcyI6WyJ1c2VyIl19LCJzY29ycGlvIjp7InJvbGVzIjpbIkZhY3RvcnktQWRtaW4iXX0sImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyJdfX0sInNjb3BlIjoib2lzcC1mcm9udGVuZCBhY2NvdW50cyBlbWFpbCBvZmZsaW5lX2FjY2VzcyB0eXBlIHByb2ZpbGUiLCJzaWQiOiJlMzFlZjA0Zi1hZjViLTRhYmQtOGYxNi1iZGE1ZmQ4YmE4NjgiLCJlbWFpbF92ZXJpZmllZCI6ZmFsc2UsIm5hbWUiOiJKb2huIERvZSIsInByZWZlcnJlZF91c2VybmFtZSI6InJlYWxtX3VzZXIiLCJhY2NvdW50cyI6W10sImdpdmVuX25hbWUiOiJKb2huIiwidHlwZSI6InVzZXIiLCJmYW1pbHlfbmFtZSI6IkRvZSIsImVtYWlsIjoidXNlckBleGFtcGxlLmNvbSJ9.DRdVljzYZAwamBhetbdKQRxVYioA8-w0D-Yzr3UdwvXynLK3mEj57N4-9FRUHUy6fpIU4BviR6-qka5WgArukBNTc20O2-AkddL4CethhYu-NyXOK0CVBqLIjmqXll-jYUiC3fGuQ1Z0ojPcr9K_Zy_VqIAfFNyuSVOaXY-H460MCP2pgjqbUhp2UezEUmttUCFzmyHwxjCYdfsjH8OAvwo8_Lglb5jwyU2wmgbXNIhOQAhhFQUWM_sibjoDvmK73bfULXkLHyqZ36AhfCpQaW-xf9X0KUyDaonA1GrN0uTjfXWFJpJ1aDVlSyWWfPKCmHk0uFFX1MOmmYDytzOKUw
Content-Type: application/json
Link: <https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.3.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"
X-Forwarded-For: 10.42.0.0
X-Forwarded-Host: ngsild.local
X-Forwarded-Port: 80
X-Forwarded-Proto: http
X-Forwarded-Server: traefik-64b96ccbcd-8v5wm
X-Real-Ip: 10.42.0.0
content-length: 345
host: host.k3d.internal:8080



Body:
{
  "id": "ngsildbroker:notification:-8660595533520922896",
  "type": "Notification",
  "subscriptionId": "urn:ngsi-ld:Subscription:17",
  "notifiedAt": "2023-04-28T13:07:01.615000Z",
  "data":   [ {
    "id" : "urn:ngsi-ld:Vehicle:A135",
    "type" : "Vehicle",
    "brandName" : {
      "type" : "Property",
      "value" : "BMW"
    }
  } ]
}

172.22.0.3 - - [28/Apr/2023 13:07:01] "POST / HTTP/1.1" 200 -
INFO:root:POST request,
Path: /
Headers:
user-agent: Vert.x-WebClient/4.2.7
Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJCYWZhVHdwMzFFMWl6RWdLUmItSGpaSWp2VDBRenZ1MmV3MjBnbE9abFJVIn0.eyJleHAiOjE2ODI2ODgxMTgsImlhdCI6MTY4MjY4NDUxOCwianRpIjoiMjZmMWJjYTYtMjkyOC00MTE0LWFmZTUtNDdkZTc2MmY1Y2MyIiwiaXNzIjoiaHR0cDovL2tleWNsb2FrLmxvY2FsL2F1dGgvcmVhbG1zL2lmZiIsImF1ZCI6WyJvaXNwLWZyb250ZW5kIiwicmVhbG0tbWFuYWdlbWVudCIsImFjY291bnQiXSwic3ViIjoiNTQyZjkyYmItNmY3Yy00ODVjLWJiMjEtOGUzZGExZWIxZDg3IiwidHlwIjoiQmVhcmVyIiwiYXpwIjoic2NvcnBpbyIsInNlc3Npb25fc3RhdGUiOiJlMzFlZjA0Zi1hZjViLTRhYmQtOGYxNi1iZGE1ZmQ4YmE4NjgiLCJyZWFsbV9hY2Nlc3MiOnsicm9sZXMiOlsib2ZmbGluZV9hY2Nlc3MiXX0sInJlc291cmNlX2FjY2VzcyI6eyJyZWFsbS1tYW5hZ2VtZW50Ijp7InJvbGVzIjpbIm1hbmFnZS11c2VycyJdfSwib2lzcC1mcm9udGVuZCI6eyJyb2xlcyI6WyJ1c2VyIl19LCJzY29ycGlvIjp7InJvbGVzIjpbIkZhY3RvcnktQWRtaW4iXX0sImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyJdfX0sInNjb3BlIjoib2lzcC1mcm9udGVuZCBhY2NvdW50cyBlbWFpbCBvZmZsaW5lX2FjY2VzcyB0eXBlIHByb2ZpbGUiLCJzaWQiOiJlMzFlZjA0Zi1hZjViLTRhYmQtOGYxNi1iZGE1ZmQ4YmE4NjgiLCJlbWFpbF92ZXJpZmllZCI6ZmFsc2UsIm5hbWUiOiJKb2huIERvZSIsInByZWZlcnJlZF91c2VybmFtZSI6InJlYWxtX3VzZXIiLCJhY2NvdW50cyI6W10sImdpdmVuX25hbWUiOiJKb2huIiwidHlwZSI6InVzZXIiLCJmYW1pbHlfbmFtZSI6IkRvZSIsImVtYWlsIjoidXNlckBleGFtcGxlLmNvbSJ9.DRdVljzYZAwamBhetbdKQRxVYioA8-w0D-Yzr3UdwvXynLK3mEj57N4-9FRUHUy6fpIU4BviR6-qka5WgArukBNTc20O2-AkddL4CethhYu-NyXOK0CVBqLIjmqXll-jYUiC3fGuQ1Z0ojPcr9K_Zy_VqIAfFNyuSVOaXY-H460MCP2pgjqbUhp2UezEUmttUCFzmyHwxjCYdfsjH8OAvwo8_Lglb5jwyU2wmgbXNIhOQAhhFQUWM_sibjoDvmK73bfULXkLHyqZ36AhfCpQaW-xf9X0KUyDaonA1GrN0uTjfXWFJpJ1aDVlSyWWfPKCmHk0uFFX1MOmmYDytzOKUw
Content-Type: application/json
Link: <https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.3.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"
X-Forwarded-For: 10.42.0.0
X-Forwarded-Host: ngsild.local
X-Forwarded-Port: 80
X-Forwarded-Proto: http
X-Forwarded-Server: traefik-64b96ccbcd-8v5wm
X-Real-Ip: 10.42.0.0
content-length: 345
host: host.k3d.internal:8080



Body:
{
  "id": "ngsildbroker:notification:-8199798479092133156",
  "type": "Notification",
  "subscriptionId": "urn:ngsi-ld:Subscription:17",
  "notifiedAt": "2023-04-28T13:07:02.853000Z",
  "data":   [ {
    "id" : "urn:ngsi-ld:Vehicle:A135",
    "type" : "Vehicle",
    "brandName" : {
      "type" : "Property",
      "value" : "BMW"
    }
  } ]
}

172.22.0.3 - - [28/Apr/2023 13:07:02] "POST / HTTP/1.1" 200 -

```
I am running the 3.0.8 Version of Scorpio All in One Runner with the in-memory Profile and the Quarkus app is running inside of a k3d cluster, therefore I am using host.k3d.internal as the host. Is this desired and if so can it be disabled? 

Edit: For the record, I have used your [tutorial](https://scorpio.readthedocs.io/en/latest/mqtt.html) here to create the entity and the subscription, only changing the Subscription uri from a mqtt to a http one.

**Comment by amitrnec** [comment]
Hi,
We are working on this issue. we will fix asap.

Thanks.

**Comment by amitrnec** [comment]
Hi,
We have fixed this issue. Fix code will be in next release.

Thanks.

**Comment by MericFeyz** [comment]
Hi, 

thanks for the heads-up! Can you link me the branch you have put the fix on, so that I can use the fix until the next release is available? Thanks!

**Comment by ScorpioBroker** [comment]
fixed in 4.0.3
